src/dash-table/derived/cell/contents.tsx

```diff
@@ -26,6 +26,7 @@ const mapRow = R.addIndex<IColumn, JSX.Element>(R.map);
 
 enum CellType {
     Dropdown,
+    DropdownLabel,
```

Comment:
To make it easier to distinguish between the general label case and the dropdown one. Processing is different.